src/Proto/nodeguard.proto

```diff
@@ -183,4 +199,44 @@ message CloseChannelRequest {
 
 // A successful response is an empty message and does NOT indicate that the channel has been closed, external monitoring is required
 message CloseChannelResponse {
-}
\ No newline at end of file
+}
+
+message GetChannelOperationRequestByIdRequest {
+  // Whether to include requests that are not managed by the NG
+  int32 channel_operation_request_id = 1;
+}
+
+enum CHANNEL_OPERATION_STATUS {
+  NULL = 0;
```

Comment:
@markettes We dont need a null = 0 enum, they will never be 0.

Reply:
It is just for easing the change from nodeguards enum and also be able to detect not set variables on the request